src/Governor.test.ts

```diff
@@ -1,9 +1,35 @@
 import axios from "axios";
 import { AxiosClient } from "./APIClient";
-import { Governor } from "./Governor";
+import { ChainId } from "./ChainId";
+import { Governor, NodeConfiguration, NodeStatus, NotionalLimit } from "./Governor";
+
 jest.mock("axios");
 const mocked = axios as jest.Mocked<typeof axios>;
 
+const data = [
+  {
+    id: "f93124b7c738843cbb89e864c862c38cddcccf95",
+    createdAt: "2022-12-15T18:37:05.751Z",
+    updatedAt: "2023-02-06T20:48:53.309Z",
+    nodename: "P2P Validator",
+    counter: 2,
+    chains: [
+      {
+        chainid: 5,
```

Comment:
We need to make sure if it is `chainid` or `chainId`
```suggestion
        chainId: 5,
```

Reply:
I'll double check, the API for some resources return chainid and for other returns chainId, I am normalizing it to chainId. Will review the test to be sure it is aligned to the current API response (staging API)